imports/plugins/core/ui/client/components/table/sortableTable.js

```diff
@@ -123,7 +127,7 @@ class SortableTable extends Component {
 
     // Add minWidth = undefined to override 100px default set by ReactTable
     const displayColumns = columnMetadata.map((element) => {
-      return _.assignIn({}, element, {
+      return _.extend({}, element, {
```

Comment:
so `assignIn` was correct here. `extend` is just an alias to `assignIn` https://lodash.com/docs/4.17.4#assignIn